Sources/Apodini/Request/Dynamics.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Dynamics.swift
+//  
+//
+//  Created by Max Obermeier on 10.12.20.
+//
+
+import Foundation
+
+public struct Dynamics {
+
+    internal var elements: [String: Any]
+    
+    init(_ elements: (name: String, Any)...) {
```

Comment:
I'd add an initializer from a dictionary, and conform to `ExpressibleByDictionaryLiteral`

Reply:
done